src/core/sprites/Sprite.js

```diff
@@ -490,12 +490,12 @@ Sprite.prototype._renderCanvas = function (renderer)
         {
             renderer.context.drawImage(
                 texture.baseTexture.source,
-                texture.crop.x * resolution,
-                texture.crop.y * resolution,
+                texture.crop.x * resolution * renderer.resolution,
```

Comment:
I'm confused by this. If this is the proper calculation then shouldn't it just be `x * texture.baseTexture.resolution`?

Right now `resolution` is defined as:

```js
var resolution = texture.baseTexture.resolution / renderer.resolution;
```

And then you are multiplying by `renderer.resolution`. Sounds like what should happen is that `resolution` should be just `var resolution = texture.baseTexture.resolution;` and all these multiplications become `* resolution`.